src/common/errors.js

```diff
@@ -1,7 +1,18 @@
 'use strict';
 const util = require('util');
 
-class RippleError extends Error {
+// this is needed because extending builtins doesn't work in babel 6.x
+function extendableBuiltin(cls) {
+  function ExtendableBuiltin() {
+    cls.apply(this, arguments);
+  }
+  ExtendableBuiltin.prototype = Object.create(cls.prototype);
+  Object.setPrototypeOf(ExtendableBuiltin, cls);
+
+  return ExtendableBuiltin;
+}
+
+class RippleError extends extendableBuiltin(Error) {
```

Comment:
What variant doesn't work under v4?
`extendableBuiltin(Error)` was added for node 0.12 support
For browsers support, actually, there will be more hack :smile: